chevah/brink
# [#3103] Initial support for wheels.

Scope
=====

Wheels are much faster than source dist and they will allow us to deploy CPU/OS specific packages

Changes
=======

Change pip to prefer wheels.

We are using old pip version... so things are still a bit slow. I hope that later we can update to latest pip and setuptools... to upgrade them we would need to check each OS... so they should be updated in a separate branch... there is already such a branch... but I need to continue to work on it

setup.py publish was also updated to upload a wheel

For brink we will still keep both tar.gz and wheel to allow older versions without wheel support to upgrade.

For all other packages we should just upload wheels.


How to test
========

reviewers: @alibotean 

run running paver clean / paver deps you should see that paver 1.2.4 is dowloaded as a wheel.

I will follow up with a branch with wheel support for server
